src/templates/BillOfLading/BillOfLadingTemplate.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { screen } from "@testing-library/dom";
+import { render } from "@testing-library/react";
+import React from "react";
+import { BillOfLadingTemplate } from "./BillOfLadingTemplate";
+import { BillOfLadingSample } from "./sample";
+jest.spyOn(HTMLCanvasElement.prototype, "getContext").mockImplementation();
+
+describe("bill of lading", () => {
```

Comment:
Maybe name it 'Bill of Lading' so can follow the other test descriptors?

Reply:
![Screenshot 2020-12-09 at 1 10 32 PM](https://user-images.githubusercontent.com/4774314/101587822-f55f6900-3a1f-11eb-88b9-c474588f711e.png)

looks like linter will complain small case